marktplaats/main.py

```diff
@@ -53,7 +53,7 @@ def get_listings(self):
                 "https://link.marktplaats.nl/" + listing["itemId"],
                 ListingImage.parse(listing.get("pictures")),
                 listing["categoryId"],
-                listing["attributes"],
+                listing["attributes"] if listing.get("attributes") else None,
```

Comment:
Could you change this to just `listing.get("attributes")`? The get method will return None anyway when the key doesn't exist.